boards/Protectli/VP66xx/v0.9.0-results.csv

```diff
@@ -0,0 +1,125 @@
+CPU runs on expected frequency (Windows),CPF002.002,PASS,PASS
+CPU with load runs on expected frequency (Ubuntu),CPF004.001,PASS,PASS
+CPU with load runs on expected frequency (Windows),CPF004.002,PASS,PASS
+Change the UUID,DCU001.001 ,PASS,PASS
+Change the serial number,DCU002.001,PASS,PASS
+Change the bootsplash logo,DCU003.001 ,PASS,PASS
+Netboot is available,NBT001.001,PASS,PASS
+OS Selection & Utilities is available,NBT002.001,PASS,PASS
+iPXE boot is available,NBT003.001,PASS,PASS
+iPXE shell is available,NBT004.001,PASS,PASS
+iPXE shell works correctly,NBT005.001,PASS,PASS
+Advanced option is available,NBT006.001,PASS,PASS
+Change netboot URL works correctly,NBT007.001,PASS,PASS
+USB detection after coldboot,UDT001.001,PASS,PASS
+Verify if no reboot occurs in the OS (Ubuntu),STB001.002,PASS,PASS
+Verify if no unexpected boot errors appear in Linux logs,STB002.001,FAIL,FAIL
+CPU clock speed displayed in setup menu,SET001.001,PASS,PASS
+RAM speed displayed in setup menu,SET002.001,PASS,PASS
+RAM size displayed in setup menu,SET003.001,PASS,PASS
+Expected CPU clock speed displayed in setup menu,SET004.001,PASS,PASS
+Expected RAM speed displayed in setup menu,SET005.001,PASS,PASS
+Expected RAM size displayed in setup menu,SET006.001,PASS,PASS
+CPU works (Ubuntu),CPU001.001,PASS,PASS
+CPU works (Windows),CPU001.002,PASS,PASS
+CPU cache enabled (Ubuntu),CPU002.001,PASS,PASS
+CPU cache enabled (Windows),CPU002.002,PASS,PASS
+Multiple CPU support (Ubuntu),CPU003.001,PASS,PASS
+Multiple CPU support (Windows),CPU003.002,PASS,PASS
+Multiple-core support (Ubuntu),CPU004.001,PASS,PASS
+Multiple-core support (Windows),CPU004.002,PASS,PASS
+Change Auto Boot Time-out and check after reboot,BMM001.001,PASS,PASS
+F9 resets Auto Boot Time-out to default value,BMM002.001,PASS,PASS
+Check Auto Boot Time-out option not accept non-numeric values,BMM003.001,PASS,PASS
+Check core count with HT disabled (Ubuntu),CCC001.001,PASS,PASS
+Check core count with HT enabled,CCC002.001,PASS,PASS
+"Check core count (HT Enabled, P: All, E: 0) (Ubuntu)",CCC003.001,PASS,PASS
+"Check core count (HT Enabled, P: All, E: All) (Ubuntu)",CCC004.001,PASS,PASS
+"Check core count (HT Disabled, P: All, E: 0) (Ubuntu)",CCC005.001,PASS,PASS
+"Check core count (HT Disabled, P: All, E: 0) (Ubuntu)",CCC006.001,PASS,PASS
+"Check core count (HT Enabled, P: 1, E: A) (Ubuntu)",CCC007.001,PASS,PASS
+"Check core count (HT Disabled, P: 1, E: A) (Ubuntu)",CCC008.001,PASS,PASS
+"Check core count (HT Enabled, P: 1, E: 1) (Ubuntu)",CCC009.001,PASS,PASS
+"Check core count (HT Disabled, P: 1, E: 1) (Ubuntu)",CCC010.001,PASS,PASS
+"Check core count (HT Enabled, P: A, E: 1) (Ubuntu)",CCC011.001,PASS,PASS
+"Check core count (HT Disabled, P: A, E: 1) (Ubuntu)",CCC012.001,PASS,PASS
+,,,
+,,,
+,,,
+,,,
+,,,
+,,,
+,,,
+,,,
+,,,
+,,,
+,,,
+,,,
+,,,
+,,,
+,,,
+,,,
+,,,
+,,,
+,,,
+,,,
+,,,
+,,,
+,,,
+,,,
+,,,
+,,,
+,,,
+,,,
+,,,
+,,,
+,,,
+,,,
+,,,
+,,,
+,,,
+,,,
```

Comment:
@philipandag delete unnecessary empty cells

Reply:
https://github.com/Dasharo/osfv-results/compare/712c7f44d1f2f034f255e421fdb95b67f7b06dac..3f146cb93a885a176c2356bb9338e4e940eb7820#diff-1b63c0b8899f3331743d48ae393b810c41e045ec5f13c867f0a1e6904d694d0cL1-R1

I missed that. It's an issue with the `update_results.py` script.